README.md

```diff
@@ -1,5 +1,6 @@
 # Massa-as-sdk
-![check-code-coverage](https://img.shields.io/badge/coverage-70%25-green)
+![check-code-coverage](https://img.shields.io/badge/coverage-69%25-red)
```

Comment:
can you add some test to avoid decreasing coverage?

Reply:
The tests are in `assembly/std/__tests__/context.spec.ts`.

Reply:
test added